sdk/python/feast/feature_service.py

```diff
@@ -104,17 +106,20 @@ def from_proto(feature_service_proto: FeatureServiceProto):
         """
         fs = FeatureService(
             name=feature_service_proto.spec.name,
-            features=[
-                FeatureViewProjection.from_proto(fp)
-                for fp in feature_service_proto.spec.features
-            ],
+            features=[],
             tags=dict(feature_service_proto.spec.tags),
             description=(
                 feature_service_proto.spec.description
                 if feature_service_proto.spec.description != ""
                 else None
             ),
         )
+        fs.feature_view_projections.extend(
```

Comment:
I'm curious why you moved this code out of the constructor. Don't feel strongly about it though.